helix-query.yaml

```
@@ -41,3122 +41,408 @@ indices:
-      lastModified:
-        select: none
-        value: |
-          parseTimestamp(headers['last-modified'], 'ddd, DD MMM YYYY hh:mm:ss GMT')
-      breadcrumbsHome:
-        select: .breadcrumbs ul li:first-child a
-        value: |
-          attribute(el, 'href')
   customer-success-stories-vn_en:
+    <<: *default
     include:
       - '/vn_en/customer-success-stories/**'
-    exclude:
-      - '**/fragments/**'
     target: /vn_en/customer-success-stories/query-index.xlsx
-    properties:
-      title:
-        select: head > meta[property="og:title"]
-        value: |
-          attribute(el, 'content')
-      date:
-        select: head > meta[name="publishdate"]
-        value: |
-          dateValue(attribute(el, 'content'), 'MM-DD-YYYY')
-      image:
-        select: head > meta[property="og:image"]
-        value: |
-          match(attribute(el, 'content'), 'https:\/\/[^/]+(\/.*)')
-      description:
-        select: head > meta[name="description"]
-        value: |
-          attribute(el, 'content')
-      topics:
-        select: head > meta[name="keywords"]
-        value: |
-          attribute(el, 'content')
-      caas-tags:
-        select: main > div > div.card-metadata > div
-        value: |
-          match(el, '[Tt]ags\n(.*)')
-      lastModified:
-        select: none
-        value: |
-          parseTimestamp(headers['last-modified'], 'ddd, DD MMM YYYY hh:mm:ss GMT')
-      breadcrumbsHome:
-        select: .breadcrumbs ul li:first-child a
-        value: |
-          attribute(el, 'href')
   customer-success-stories-vn_vi:
+    <<: *default
     include:
       - '/vn_vi/customer-success-stories/**'
-    exclude:
-      - '**/fragments/**'
     target: /vn_vi/customer-success-stories/query-index.xlsx
-    properties:
-      title:
-        select: head > meta[property="og:title"]
-        value: |
-          attribute(el, 'content')
-      date:
-        select: head > meta[name="publishdate"]
-        value: |
-          dateValue(attribute(el, 'content'), 'MM-DD-YYYY')
-      image:
-        select: head > meta[property="og:image"]
-        value: |
-          match(attribute(el, 'content'), 'https:\/\/[^/]+(\/.*)')
-      description:
-        select: head > meta[name="description"]
-        value: |
-          attribute(el, 'content')
-      topics:
-        select: head > meta[name="keywords"]
-        value: |
-          attribute(el, 'content')
-      caas-tags:
-        select: main > div > div.card-metadata > div
-        value: |
-          match(el, '[Tt]ags\n(.*)')
-      lastModified:
-        select: none
-        value: |
-          parseTimestamp(headers['last-modified'], 'ddd, DD MMM YYYY hh:mm:ss GMT')
-      breadcrumbsHome:
-        select: .breadcrumbs ul li:first-child a
-        value: |
-          attribute(el, 'href')
+  customer-success-stories-videos:
+    <<: *default
+    include:
+      - '**/fragments/customer-success-stories/featured-videos/**'
```

Should this include US or have a second index for US?
```suggestion
      - '**/fragments/customer-success-stories/featured-videos/**'
      - '/fragments/customer-success-stories/featured-videos/**'
```

---

Not necessary. It'll include US as is